docs/how-tos/load-data.rst

```diff
@@ -1,12 +0,0 @@
-==================
```

Comment:
even though this section isn't great, I think a `loading data` guide makes sense that points to examples/concepts (and even the recording from today).